.github/workflows/smoke-checks.yml

```diff
@@ -156,11 +159,11 @@ jobs:
     - uses: ./.github/actions/bootstrap
     - name: Build LLC (Xcode 12)
       env:
-        DEVELOPER_DIR: "/Applications/Xcode_12.5.1.app/Contents/Developer"
+        XCODE_VERSION: "12.5.1"
```

Comment:
Why don't we put this to env too?

Reply:
@b-onc Can you elaborate a bit more. Thanks!

Reply:
```
XCODE_VERSION: ${{ env.XCODE_VERSION }}
```

Reply:
The issue with the above proposal is that anytime we try to update our CI, we will change the Xcode version globally for all CI runs.

Reply:
gotcha. in my hurry i missed that it's Xcode 12 and not 13